src/components/pages/RoleApply/Apply.js

```diff
@@ -3,6 +3,8 @@ import './Styles/apply.css';
 import { Link } from 'react-router-dom';
 import { Card, Button, Row, Col, Space, Breadcrumb } from 'antd';
 import { LoginOutlined, IdcardOutlined } from '@ant-design/icons';
+import img1 from '../../../images/pexels-photo-7504837.png';
+import img2 from '../../../images/pexels-photo-1181337 2.png';
```

Comment:
Agree with John here - a better approach would be to remove the old version of the image and rename the new one by removing the ` 2` so that we're simply swapping the image and not introducing potential ways to trip up future devs looking through this codebase.